sql/catalyst/src/main/scala/org/apache/spark/sql/Encoders.scala

```diff
@@ -140,7 +140,7 @@ object Encoders {
    * Creates an encoder that serializes instances of the `java.time.Duration` class
    * to the internal representation of nullable Catalyst's DayTimeIntervalType.
    *
-   * @since 3.2.0
+   * @since 3.3.0
```

Comment:
Please, revert it back. This is related to interval types.

Reply:
ok, there is something wrong during cherry pick